### DISCUSSION TITLE
Is it possible to disable runtime nunjucks-slim if you only need server side templating?

### DISCUSSION THREAD

**Discussion by victorpoughon** [ogonkov/nunjucks-loader#105] [General]
I'm trying to use this loader for static templates server side. I don't need any runtime templating. I'm getting this error:

```
ERROR in   Error: Child compilation failed:
  Module not found: Error: Can't resolve 'nunjucks/browser/nunjucks-slim'
```

I have only installed nunjucks with `npm install -g nunjucks` and the plugin. Can I disable all runtime stuff somehow?

**Comment by ogonkov** [comment]
Nunjucks is still required to create `Environment` that will render your precompiled template. Looks like webpack couldn't find global nunjucks.

Can you share your webpack config and what is your server render looks like?

**Comment by victorpoughon** [comment]
Nevermind, I got it to work after reinstalling everything. Thanks for a great package, it seems to work great.

**Reply by victorpoughon** [reply]
I think this time I install nunjucks locally and not globally, that's propably it.
